app/src/main/java/com/android/partagix/model/BorrowViewModel.kt

```diff
@@ -0,0 +1,90 @@
+package com.android.partagix.model
+
+import android.location.Location
+import androidx.lifecycle.ViewModel
+import com.android.partagix.model.category.Category
+import com.android.partagix.model.inventory.Inventory
+import com.android.partagix.model.item.Item
+import com.android.partagix.model.loan.Loan
+import com.android.partagix.model.loan.LoanState
+import com.android.partagix.model.user.User
+import com.android.partagix.model.visibility.Visibility
+import java.sql.Date
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.StateFlow
+
+val emptyLoan = Loan("", "", "", "", Date(0), Date(0), "", "", "", "", LoanState.PENDING)
```

Comment:
there empty variables already exists, in the model/emptyConst/empty.kt

Reply:
Nice I didn't know. I will now use them.